task0/PythonIntro-Batuhan.ipynb

```diff
@@ -0,0 +1,2362 @@
+{
```

Comment:
<p>do we need it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bogaziciai/assignment/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>